### PR TITLE
Add VBank Coin

### PR DESCRIPTION
**Pull request by V-Bank**

### CODE REVIEW

**Comment by prusnak** [comment]
use the first available value
use the lowercase letters for hex

**Comment by prusnak** [comment]
- use the first available value

File: slip-0044.md [hunk 1]
```diff
@@ -1086,6 +1086,7 @@ index | hexa       | symbol | coin
 4242  | 0x80001092 | AXE    | [Axe](https://github.com/AXErunners/axe)
 5248  | 0x00001480 | FIC    | [FIC](https://ficnetwork.com)
 5353  | 0x000014e9 | HNS    | [Handshake](https://handshake.org)
+5555  | 0x800015b3 | VBK    | [V-Bank](http://www.vbank.group/)
```
Comment (jutarad) [inline slip-0044.md]: tjutarad@gmail.com
```suggestion
5555  | 0x800015b3 | VBK    | [V-Bank](http://www.vbank.group/)
```

Comment (jutarad) [inline slip-0044.md]: tjutarad@gmail.com

Comment (jutarad) [inline slip-0044.md]:
```suggestion
5555  | 0x800015b3 | VBK    | [V-Bank](http://www.vbank.group/)
```

**Comment by prusnak** [comment]
Open a new pull request

**Comment by V-Bank** [comment]
what's the matter?

**Pull request closed without merging**